utils/utils.go

```diff
@@ -27,7 +27,8 @@ import (
 )
 
 const (
-	DotDelimiter = "."
+	DotDelimiter          = "."
+	WFActivityIDDelimiter = "::" // Delimiter between workflow id and activity id
```

Comment:
I think this const would be better to live in `pkg/runtime/wfengine`, or ideally, in `pkg/runtime/consts` (I am currently in the process of breaking up `runtime.go` where I have also created this package).